<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cybersport News</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {

	color: rgb(200, 200, 190);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(1, 5, 59, 255));

}
QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(15, 15, 15, 255), stop:1 rgba(80, 80, 80, 255));
}

QPushButton:pressed {
	background-color: rgb(120, 120, 120)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:40pt;
border:none;</string>
      </property>
      <property name="text">
       <string>CYBERINFO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btn_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:30pt;
border:1px solid rgb(76, 27, 11);</string>
        </property>
        <property name="text">
         <string>CS:GO News</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:30pt;
border:1px solid rgb(76, 27, 11);</string>
        </property>
        <property name="text">
         <string>DOTA 2 News</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:30pt;
border:1px solid rgb(76, 27, 11);</string>
        </property>
        <property name="text">
         <string>CS:GO Training</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:30pt;
border:1px solid rgb(76, 27, 11);</string>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
